app/scripts/components/common/blocks/scrollytelling/index.tsx

```diff
@@ -48,14 +49,15 @@ export const scrollyMapHeight = 'calc(100vh - 3rem)';
 
 const ScrollyMapWrapper = styled.div``;
 
-const TheMap = styled.div`
+const TheMap = styled.div<{ topOffset: number }>`
   height: ${scrollyMapHeight};
   position: sticky;
-  top: 3rem;
+  transition: top ${HERO_TRANSITION_DURATION}ms ease-out,
+    height ${HERO_TRANSITION_DURATION}ms ease-out;
 
-  ${media.mediumUp`
-    height: calc(100vh - 4rem);
-    top: 4rem;
+  ${({ topOffset }) => css`
+    top: ${topOffset}px;
+    height: calc(100vh - ${topOffset}px);
```

Comment:
Is that the equivalent of the following?:
```
top: ${({ topOffset }) => topOffset}px;
height: calc(100vh - ${({ topOffset }) => topOffset}px);
```

Not saying it's any better, but the [styled components doc](https://styled-components.com/docs/api#css) makes it confusing to me